File: Sep_week2/주식가격/hyeonbin.cs
using System;

namespace ConsoleApp1
{
    public class Solution
    {
        public int[] solution(int[] prices)
        {
            int[] answer = new int[prices.Length];
            int fall;

            for (int i = 0; i < prices.Length - 1; i++)
            {
                fall = 0;

                for (int j = i + 1; j < prices.Length; j++)
                {
                    fall++;
                    if (prices[i] > prices[j])
                    {
                        break;
                    }

                }
                answer[i] = fall;
            }

            return answer;
        }
    }
}

File: Sep_week4/문자열 압축/Hyeonbin.py
def solution(s):
    answer = []
    com_line = ""
    answer.append(len(s))

    for length in range(1, len(s) // + 1):
        line = ""
        num = 1
        for i in range(0, len(s), length):
            if s[i:i + length] == s[i + length:i + (2 * length)]:
                num += 1
            else:
                if num != 1:
                    line += str(num) + s[i:i + length]
                    com_line += line
                    num = 1
                    line = ""
                else:
                    com_line += s[i:i + length]

        if len(com_line) != len(s):
            answer.append(len(com_line))
            print(com_line)
        com_line = ""

    return min(answer)